newton/tests/builders/testing.py

```
@@ -0,0 +1,1839 @@
+_SHAPE_EXPECTED_LEN: dict[GeoType, int] = {
+    GeoType.SPHERE: 1,
+    GeoType.CYLINDER: 2,
+    GeoType.CONE: 2,
+    GeoType.CAPSULE: 2,
+    GeoType.BOX: 3,
+    GeoType.ELLIPSOID: 3,
+    GeoType.PLANE: 4,
+}
+
+
+def make_shape_initial_position(type: GeoType, dims: tuple, is_top: bool = True) -> wp.vec3:
```

Ruff (0.15.11)

[error] Function argument `type` is shadowing a Python builtin

(A002)